lib/index.html

```diff
@@ -27,16 +26,16 @@
 		<div class="wrapper left">
 			<div class="hero-links">
 				<a href="https://github.com/meetalva/alva" target="_blank" class="hero-github" title="Alva on Github">
-					<img src="images/github.svg" role="presentation">
+					<img src="images/github.svg" role="presentation" alt="">
```

Comment:
alt text could be `Github logo`

Reply:
Yes – but directly next to it is a text saying "Alva on Github"?

Reply:
added an empty alt="" tag. a screenreader saying "Github logo; Alva on Github" doesn’t make so much sense, does it?

Reply:
sounds good